message/infrastructure/sql/publisher.go

```diff
@@ -38,6 +43,7 @@ func (c *PublisherConfig) setDefaults() {
 // db is implemented both by *sql.DB and *sql.Tx
 type db interface {
 	Prepare(q string) (*sql.Stmt, error)
```

Comment:
If all statements are removed,, `Prepare` is not needed anymore in this interface?

Reply:
removed